prometheus.yml

```
@@ -0,0 +1,13 @@
+  scrape_interval: 5s  # 5초마다 데이터 수집
+
+scrape_configs:
+  - job_name: 'spring-boot'
+    metrics_path: '/actuator/prometheus'
+    static_configs:
+      - targets:
+          - 'host.docker.internal:8081'
+          - 'host.docker.internal:8082'
+          - 'host.docker.internal:8083'
+          - 'host.docker.internal:8084'
+          - 'host.docker.internal:8085'
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)